Assets/HoloToolkit/Input/Scripts/CursorManager.cs

```diff
@@ -1,65 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
It is not sentimental. It is ensuring you are actually helping developers along the way.